pkg/executor/test/executor/executor_test.go

```diff
@@ -2724,3 +2724,36 @@ func TestProcessInfoOfSubQuery(t *testing.T) {
 	tk2.MustQuery("select 1 from information_schema.processlist where TxnStart != '' and info like 'select%sleep% from t%'").Check(testkit.Rows("1"))
 	wg.Wait()
 }
+
+func TestIssues49377(t *testing.T) {
```

Comment:
Could you change this to the MySQL-tester based test? （https://github.com/pingcap/tidb/tree/master/tests/integrationtest）

Reply:
i‘ll leave it to next pr